utils/buildstock_releases.json

```diff
@@ -0,0 +1,394 @@
+{
+  "com_2021_amy2018_1": {
+    "release_year": "2021",
+    "res_com": "comstock",
+    "weather": "amy2018",
+    "release_number": "1",
+    "upgrade_ids": []
+  },
+  "com_2021_tmy3_1": {
+    "release_year": "2021",
+    "res_com": "comstock",
+    "weather": "tmy3",
+    "release_number": "1",
+    "upgrade_ids": []
+  },
+  "res_2021_amy2018_1": {
+    "release_year": "2021",
+    "res_com": "resstock",
+    "weather": "amy2018",
+    "release_number": "1",
+    "upgrade_ids": []
+  },
+  "res_2021_tmy3_1": {
+    "release_year": "2021",
+    "res_com": "resstock",
+    "weather": "tmy3",
+    "release_number": "1",
+    "upgrade_ids": []
+  },
+  "res_2022_amy2012_1.1": {
+    "release_year": "2022",
+    "res_com": "resstock",
+    "weather": "amy2012",
+    "release_number": "1.1",
+    "upgrade_ids": [
+      "0",
+      "1",
+      "2",
+      "3",
+      "4",
+      "5",
+      "6",
+      "7",
+      "8",
+      "9",
+      "10"
+    ]
+  },
+  "res_2022_amy2012_1": {
+    "release_year": "2022",
+    "res_com": "resstock",
+    "weather": "amy2012",
+    "release_number": "1",
+    "upgrade_ids": [
+      "0",
+      "1",
+      "2",
+      "3",
+      "4",
+      "5",
+      "6",
+      "7",
+      "8",
+      "9",
+      "10"
+    ]
+  },
+  "res_2022_amy2018_1.1": {
+    "release_year": "2022",
+    "res_com": "resstock",
+    "weather": "amy2018",
+    "release_number": "1.1",
+    "upgrade_ids": [
+      "0",
+      "1",
+      "2",
+      "3",
+      "4",
+      "5",
+      "6",
+      "7",
+      "8",
+      "9",
+      "10"
+    ]
+  },
+  "res_2022_amy2018_1": {
+    "release_year": "2022",
+    "res_com": "resstock",
+    "weather": "amy2018",
+    "release_number": "1",
+    "upgrade_ids": [
+      "0",
+      "1",
+      "2",
+      "3",
+      "4",
+      "5",
+      "6",
+      "7",
+      "8",
+      "9",
+      "10"
+    ]
+  },
+  "res_2022_tmy3_1.1": {
+    "release_year": "2022",
+    "res_com": "resstock",
+    "weather": "tmy3",
+    "release_number": "1.1",
+    "upgrade_ids": [
+      "0",
+      "1",
+      "2",
+      "3",
+      "4",
+      "5",
+      "6",
+      "7",
+      "8",
+      "9",
+      "10"
+    ]
+  },
+  "res_2022_tmy3_1": {
+    "release_year": "2022",
+    "res_com": "resstock",
+    "weather": "tmy3",
+    "release_number": "1",
+    "upgrade_ids": [
+      "0",
+      "1",
+      "2",
+      "3",
+      "4",
+      "5",
+      "6",
+      "7",
+      "8",
+      "9",
+      "10"
+    ]
+  },
+  "com_2023_amy2018_1": {
+    "release_year": "2023",
+    "res_com": "comstock",
+    "weather": "amy2018",
+    "release_number": "1",
+    "upgrade_ids": [
+      "00",
+      "01",
+      "02",
+      "03",
+      "04",
+      "05",
+      "06",
+      "07",
+      "08",
+      "09",
+      "10"
+    ]
+  },
+  "com_2023_amy2018_2": {
```

Comment:
@alexhyunminlee What changed between release 2 and 1? Is release 1 valid or just buggy?